### PR TITLE
Update resolvers to fetch dependencies via TLS

### PR DESCRIPTION
**Pull request by mio-stripe**
Do-over of https://github.com/stripe/rainier/pull/337 following @beala-stripe 's suggestion https://github.com/stripe/rainier/pull/337#issuecomment-478730807 to update the resolvers rather than editing the WORKSPACE file by hand.

### CODE REVIEW

**Comment by mio-stripe** [comment]
@beala-stripe mind giving me some pointers in rerunning sbt-bazel so I can make sure this works?

**Comment by beala-stripe** [comment]
`bazelGenerate` is the sbt task that generates the bazel files.

You can:
```
cd rainier
sbt bazelGenerate
```

Or run the sbt repl and run `bazelGenerate` inside it.

**Comment by mio-stripe** [comment]
Sweet, thanks!
r? @avi-stripe @beala-stripe

**Comment by avibryant** [comment]
LGTM

File: rainier-core/BUILD [hunk 1]
```diff
@@ -8,15 +8,12 @@ load(
 scala_library(
   name = 'rainierCore',
   deps = [
-    '//.rainier-shaded-asm:asmShaded',
-    '//.rainier-shaded-asm:asmTreeShaded',
+    '//external:jar/com/google/code/findbugs/jsr305',
     '//external:jar/com/google/flogger/flogger',
-    
-  ],
-  runtime_deps = [
```
Comment (beala-stripe) [inline rainier-core/BUILD]: It looks like these runtim_deps got dropped. Is it possible this file was edited by hand before?

Comment (mio-stripe) [reply rainier-core/BUILD]: whoa, good catch. Dunno, but I'll just add them back.

Comment (mio-stripe) [reply rainier-core/BUILD]: Running `sbt bazelGenerate` on current develop, I get https://github.com/stripe/rainier/compare/mio/no-https which also deletes those deps. I know shaded-asm has some complicated history, did we intentionally change this? (cc: @avi-stripe)

**Comment by nati-stripe** [comment]
Thanks for tackling this the correct way @mio-stripe @beala-stripe !

**Pull request merged**